cpp/Platform.Timestamps/Timestamp.h

```diff
@@ -1,25 +1,73 @@
-﻿namespace Platform::Timestamps
-{
-    struct Timestamp : public IEquatable<Timestamp>
-    {
-        public: inline static std::string DefaultFormat = "yyyy.MM.dd hh:mm:ss.fffffff";
-
-        public: std::uint64_t Ticks = 0;
+﻿#pragma once
 
-        public: Timestamp(std::uint64_t ticks) { Ticks = ticks; }
+#include "CommonEraClock.h"
+#include <functional>
+#include <iomanip>
+#include <sstream>
+#include <string>
 
-        public: Timestamp(DateTime dateTime) : Timestamp((std::uint64_t)dateTime.ToUniversalTime().Ticks) { }
-
-        public: operator DateTime() const { return DateTime((std::int64_t)this->Ticks, DateTimeKind.Utc); }
+namespace Platform::Timestamps
+{
+    class Timestamp
+    {
+    public:
+        static constexpr const char* DefaultFormat = "%Y.%m.%d %H:%M:%S";
+        static constexpr int TicksPerSecond = 10000000;
 
-        public: Timestamp(std::uint64_t ticks) : Timestamp(ticks) { }
+        Timestamp(const uint64_t& ticks)
```

Comment:
pls by value and a little more words can be written
```cpp
explicit constexpr TimeStamp(std::uint64_t ticks) noexcept
```
explicit for type-safe, constexpr because it really can be calculated in compile time and noexcept, because it realy not throws